src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -123,13 +125,13 @@ class DeBruijnGraph(val kmerSize: Int,
   }
 
   /**
-   *  Searches forward or backward from a node to find those connected by a unique path
+    *  Searches forward or backward from a node to find those connected by a unique path
```

Comment:
Definitely a job for intellij, just need to decide what to tell it we want.

In other news, github seems completely confused/wrong about what the diff is here; I see that your underlying code has everything aligned to first asterisk and yet github showing me

![](http://cl.ly/3y0B3J352p3n/Screen%20Shot%202016-05-03%20at%2012.09.48%20PM.png)

![](http://cl.ly/1C1j3N3V1d3f/Screen%20Shot%202016-05-03%20at%2012.11.50%20PM.png)

wat.

anyway, this is all resolved here, thx